### PR TITLE
lower zookeeper session timeout

### PR DESCRIPTION
**Pull request by adyach**
# One-line summary
Lower zookeeper session timeout

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java [hunk 1]
```diff
@@ -29,18 +30,25 @@
...
-                                  final KafkaTopicConfigFactory kafkaTopicConfigFactory,
-                                  final MetricRegistry metricRegistry) {
+    public KafkaRepositoryCreator(
+            final NakadiSettings nakadiSettings,
+            final KafkaSettings kafkaSettings,
+            final ZookeeperSettings zookeeperSettings,
+            final KafkaTopicConfigFactory kafkaTopicConfigFactory,
+            final MetricRegistry metricRegistry,
+            @Value("${nakadi.zookeeper.exhibitor.session-timeout-ms:10000}") final Integer sessionTimeoutMs,
+            @Value("${nakadi.zookeeper.exhibitor.connection-timeout-ms:3000}") final Integer connectionTimeoutMs) {
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: Can you please move the default values to application.yml ?

Comment (adyach) [reply src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: no, it is already quite big, I do not want to move there every parameter of the system

Comment (v-stepanov) [reply src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: The current situation is that you duplicated this in two places

Comment (adyach) [reply src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: ok

Comment (adyach) [reply src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: ahaha https://github.com/zalando/nakadi/blob/master/src/main/resources/application.yml#L75

Comment (adyach) [reply src/main/java/org/zalando/nakadi/repository/KafkaRepositoryCreator.java]: @v-stepanov fixed, thanks

**Comment by v-stepanov** [comment]
👍

**Comment by adyach** [comment]
👍

**Pull request merged**